handler/pair.go

```diff
@@ -0,0 +1,47 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+	"path"
+
+	"github.com/gorilla/mux"
+
+	"github.com/cloudradar-monitoring/plexus/api"
+)
+
+// Pair godoc
+// @Summary Gets the pairing code info.
+// @Tags session
+// @Produce application/json
+// @Param code path string true "code"
+// @Security BasicAuth
+// @Success 200 {object} api.URLResponse
+// @Failure 401 {object} api.Error
+// @Failure 404 {object} api.Error
+// @Router /session/{code} [get]
+func (h *Handler) Pair(rw http.ResponseWriter, r *http.Request) {
+	h.lock.Lock()
+	defer h.lock.Unlock()
+	id, ok := h.codes[mux.Vars(r)["id"]]
+	if !ok {
+		api.WriteJSONResponse(rw, http.StatusNotFound, map[string]interface{}{
+			"message": "Code not found",
+		})
+		return
+	}
+
+	session, ok := h.checkSessionAuthentication(rw, r, id)
+	if !ok {
+		return
+	}
+
+	api.WriteJSONResponse(rw, http.StatusNotFound, api.PairedSession{
```

Comment:
this should be status ok.

Reply:
why status ok?

Reply:
Because the request status should be a success and not a client error like not found?

Reply:
sorry my bad I taught you wanted the other code on 28 line to be status ok I didn't even look at this code